src/Apps/MyCollection/Routes/MyCollectionArtwork/Components/MyCollectionArtworkComparables.tsx

```diff
@@ -0,0 +1,79 @@
+import { Column, Flex, Join, Message, Spacer, Text } from "@artsy/palette"
+import { createFragmentContainer, graphql } from "react-relay"
+import { MetaTags } from "Components/MetaTags"
+import { ArtistAuctionResultItemFragmentContainer } from "Apps/Artist/Routes/AuctionResults/ArtistAuctionResultItem"
+import { extractNodes } from "Utils/extractNodes"
+import { __internal__useMatchMedia } from "Utils/Hooks/useMatchMedia"
+import { MyCollectionArtworkComparables_artwork } from "__generated__/MyCollectionArtworkComparables_artwork.graphql"
+interface MyCollectionArtworkComparablesProps {
+  artwork: MyCollectionArtworkComparables_artwork
+}
+
+const ComparablesContainer: React.FC<MyCollectionArtworkComparablesProps> = ({
```

Comment:
**(suggestion):** I think, as a pattern, we always name the file the same as the component.

```suggestion
const MyCollectionArtworkComparables: React.FC<MyCollectionArtworkComparablesProps> = ({
```